frame/executive/src/lib.rs

```diff
@@ -303,7 +305,8 @@ where
 		}
 	}
 
-	fn initial_checks(block: &Block) {
+	/// Checks the block is valid, and returns the index of the first signed extrinsic in the block.
```

Comment:
```suggestion
	/// Checks the block is valid, and returns the index of the first non-inherent in the block.
```
This is more technically correct; we return the index of the first unsigned/signed extrinsic 

relates to https://github.com/paritytech/substrate/pull/9324/files?file-filters%5B%5D=.rs#r667524538